Tests/WorkspaceTests/WorkspaceStateTests.swift

```diff
@@ -0,0 +1,85 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import TSCBasic
+@testable import Workspace
+import XCTest
+
+final class WorkspaceStateTests: XCTestCase {
+    private var fs: FileSystem! = nil
+
+    override func setUp() {
+        super.setUp()
+
+        fs = InMemoryFileSystem()
```

Comment:
probably do this at a test by test basis, as we dont always use the in-memory one